org.idempiere.test/src/org/idempiere/test/adwindow/GridTabTest.java

```diff
@@ -250,6 +250,8 @@ public void dataStatusChanged(DataStatusEvent e) {
 							if (gridTab.getAD_Table_ID() != e.AD_Table_ID)
 								throw new RuntimeException("Table doesn't match with updated tab");
 						}
+						if (gridTab == null)
+							throw new RuntimeException("Tab not found for table " + e.AD_Table_ID);
```

Comment:
this is in unit test, if gridTab == null is unexpected, we should just fail it using unit test method like fail.